src/elements/content-sidebar/activity-feed/common/activity-error/ActivityError.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../../../../styles/variables';
+
+.bcs-ActivityError {
+    padding: 14px 15px;
+}
+
+.bcs-ActivityError-action,
+button.bcs-ActivityError-action {
```

Comment:
You can use `&` to avoid repeating the class name both here and in UserLink. Also consider using `.btn-plain` rather than an element selector.

Reply:
The `button` part is just to match specificity; ideally the classname itself would be enough. Since we have a lot of issues with `.btn-plain` I decided to just avoid it for now since we know there has to be an underlying button element.